src/ucs/sys/string.c

```diff
@@ -470,7 +470,7 @@ ucs_status_t ucs_string_alloc_formatted_path(char **buffer_p, const char *name,
 {
     va_list ap;
     ucs_status_t status;
-    char *temp_buffer;
+    char *temp_buffer = NULL;
```

Comment:
why needed to initialize?

Reply:
Because if I don't: 
![image](https://github.com/user-attachments/assets/a4bfe642-2d42-478d-a728-d383741d1509)